gateway_to_backend/protocol_buffers_files/otap_message.proto

```diff
@@ -24,11 +24,23 @@ message GetScratchpadStatusResp {
 }
 
 message UploadScratchpadReq {
+    message ChunkInfo {
+        // Id of the scratchpad this chunk belongs
+        required uint32 scratchpad_unique_id = 1;
+        // Offset of the chunk in the full scratchpad
+        // If 0, then it is the first chunk
+        required uint32 offset = 2;
+        // Is it the last chunk
+        required bool is_last_chunk = 3;
```

Comment:
As @teemupiiroinenwirepas pointed out, it is somewhat hard to envision how the flow will be implemented. However, using only this single flag seems to mandate the usage of sequential flow and sending of the chunks in order. But I guess that is anyways mandatory, if the intention of the feature is to overcome memory limitations in the gateway side.

Reply:
Yes, and after checking our bootloader code, it is anyway mandatory at this level to receive the block in order, otherwise it will fail the sequence completely.
So we cannot rely on broker behavior for managing the flow control.
So flow control must be handled between backend and gw, so next block should be sent after the previous one is acknowledged.